core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -625,6 +625,7 @@ static class LoadingTime extends MasterToSlaveCallable<Long, RuntimeException> {
      * @param listener Channel event listener to be attached (if not {@code null})
      * @since 1.444
      */
+    @SuppressFBWarnings(value = {"NN_NAKED_NOTIFY"}, justification = "False positive, the warning isn't for this scenario")
```

Comment:
Fails a checkstyle check for me locally without this change:
```suggestion
    @SuppressFBWarnings(value = "NN_NAKED_NOTIFY", justification = "False positive, the warning isn't for this scenario")
```